README.md

```diff
@@ -18,7 +18,7 @@ Rook is currently in alpha state and has focused initially on orchestrating Ceph
 
 ## Getting Started and Documentation
 
-For installation, deployment, and administration, see our [Documentation](https://rook.github.io/docs/rook/latest/kubernetes.html).
+For installation, deployment, and administration, see our [Documentation](https://rook.github.io/docs/rook/master/kubernetes.html).
```

Comment:
can this link just become `https://rook.github.io/docs/rook/master`? Maybe add the index page as part of this PR